### PR TITLE
WSL-distrod: Add version 0.1.7

### PR DESCRIPTION
**Pull request by Slach**
## WSL-distrod

Distrod is a meta-distro for WSL 2 which installs Ubuntu, Arch, Debian, Gentoo, etc. with systemd in a minute for you. Distrod also has built-in auto-start feature on Windows startup and port forwarding ability.

### CODE REVIEW

**Comment by Slach** [comment]
don't understand why 500 commits in pull request i just make 
```
git pull origin master
git checkout -b wsl-distrod
create file bucket/wsl-distrod.json
and 
git push origin wsl-distrod
```

**Comment by igitur** [comment]
Please rebase on `master` branch.

**Comment by Slach** [comment]
@igitur commits rebased and squashed
could you please review again?

File: bucket/wsl-distrod.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.1.7",
+    "description": "Distrod is a meta-distro for WSL 2 which installs Ubuntu, Arch, Debian, Gentoo, etc. with systemd in a minute for you. Distrod also has built-in auto-start feature on Windows startup and port forwarding ability.",
```
Comment (niheaven) [inline bucket/wsl-distrod.json:3]:
```suggestion
    "description": "A meta-distro for WSL 2 which installs Ubuntu, Arch, Debian, Gentoo, etc. with systemd in a minute for you",
```

File: bucket/wsl-distrod.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://github.com/nullpo-head/wsl-distrod",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/nullpo-head/wsl-distrod/releases/download/v0.1.7/distrod_wsl_launcher-x86_64.zip",
+            "hash": "168ed22170e2c38914e11fb4d6e18822dbb2c85730f86e166eb99f2927634fd2",
+            "bin": "distrod_wsl_launcher-x86_64\\distrod_wsl_launcher.exe"
```
Comment (niheaven) [inline bucket/wsl-distrod.json:10]:
```suggestion
            "extract_dir": "distrod_wsl_launcher-x86_64"
```

File: bucket/wsl-distrod.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        }
+    },
```
Comment (niheaven) [inline bucket/wsl-distrod.json:12]:
```suggestion
    },
    "bin": "distrod_wsl_launcher.exe",
```

**Review by niheaven — CHANGES_REQUESTED** [review]

**Comment by Slach** [comment]
@niheaven thanks for review, suggestions committed

**Review by niheaven — APPROVED** [review]

**Pull request merged**